packages/flame/lib/src/text/common/glyph_data.dart

```diff
@@ -1,15 +1,19 @@
+@Deprecated('This class will be removed in 1.5.0; use Glyph instead')
```

Comment:
1.6.0 right? Since 1.4.0 is released now

Reply:
Right, forgot to update these after the release.